server/models/review-request.js

```diff
@@ -77,6 +83,12 @@ const requestSchema = new mongoose.Schema({
     enum: ["pending", "declined", "accepted", "closed"],
     default: "pending",
   },
+  rating: {
+    type: Number,
+    min: 0,
+    max: 5,
+    default: 0,
+  },
```

Comment:
may not need rating in request since rating is already in review. can always access rating inside embeddedReview if needed

Reply:
I thought the same but just included it because of the wording on the ticket. 

Good point on the embedded review containing the rating.

I will fix this later tonight.